.github/ISSUE_TEMPLATE/invitation.md

```diff
@@ -2,7 +2,7 @@
 name: Invitation to the GitHub Community Organization
 about: I would like to be part of the awesome community
 title: 'Please invite me to the GitHub Community Organization'
-labels: 'invite me to the organization'
+labels: invite me to the organization
```

Comment:
Looks good but the label is in UK English spelling. 
`invite me to the organisation`